litellm/llms/base_aws_llm.py

```diff
@@ -119,8 +119,6 @@ def get_credentials(
                     "aws_web_identity_token": aws_web_identity_token,
                     "aws_role_name": aws_role_name,
                     "aws_session_name": aws_session_name,
-                    "aws_region_name": aws_region_name,
```

Comment:
oh how / where is the aws_region_name moved to? not sure i see it in the new commit

Reply:
got it